chat/utils.js

```diff
@@ -0,0 +1,39 @@
+import { isString, get } from 'lodash';
```

Comment:
I think @drewbarontini mentioned this somewhere, but I don't think we have tree shaking in the app. So, won't all of lodash get loaded here?

Reply:
Good call, there were a few other places that snuck in with this un-tree-shake friendly approach, so I fixed those too (8571d45).
Also ditched `get` altogether to use optional chaining like you suggested below.
Thanks for the tips 🙇‍♂️ .